example_app_generator/generate_stuff.rb

```diff
@@ -111,6 +111,12 @@ def using_source_path(path)
 rescue LoadError
 end
 
+begin
+  require 'action_cable'
+  generate('channel chat') if ::Rails::VERSION::STRING.to_f >= 6.0
+rescue LoadError
```

Comment:
When would this generate a load error when on Rails 6? Seems like the two gates could be combined?